### PR TITLE
Task/fp 1801 guides breadcrumbs headings

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

1. Consistent heading, breadcrumb, and `<hr>` / `<h2>`-border styling. <sub>(sources: [#apcd_dev](https://tacc-team.slack.com/archives/C03B7899YD8/p1661370303192069), _and a private request_)</sub>

2. <details><summary>Promote the use of <code>&lt;h2&gt;</code> with border instead of adding an <code>&lt;hr&gt;</code>. <sub>(click to toggle reasoning)</sub></summary>

    > _The `<hr>` HTML element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section._ — [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr)

    I.e. the `<hr>` is **not** intended to be used to create a border underneath headings.

    > _Historically, [it] has been presented as a horizontal rule or line. While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, so if you wish to draw a horizontal line, you should do so using appropriate CSS._ — [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr)

    </details>

3. Two neglected typography tweaks.

## Related

- [FP-1801]
- requires https://github.com/TACC/Core-Styles/pull/45
- required by https://github.com/TACC/Core-CMS-Resources/pull/164
- required by https://github.com/TACC/Core-CMS-Custom/pull/6

## Changes

- **[FP-1800] Integrate New Typography into Core**
- **[FP-1452] Add Breadcrumbs to Guides Pages**
- **[FP-1802] TAM-Based Getting Started Guide**
- [FP-1336] Add Lowercase Text to Typography Guide Headings
- [FP-1335]: Do Not Force Capitalize `<h4>`

<details><summary>Details <sub>(click to toggle details)</sub></summary>

- feat: install related Core-Styles branch (see [TACC/Core-Styles#45](https://github.com/TACC/Core-Styles/pull/45))
- feat: load space vars from Core-Styles **not** Core-CMS
- fix: load breadcrumbs on guide pages
- feat: tam-based Getting Started guide
- feat: change original Getting Started guide to load its renamed version
- feat: rename original Getting Started guide to v2
- fix: guide pages to start at `<h1>` **not** `<h2>`
- feat: update Typography snippet to load guide styles and show headings with and without paragraph beneath.
- feat: use lowercase text in some headings (to test casing)

</details>

### Setup

**Either** test remotely at https://apcd-qa.tacc.utexas.edu/help/ …

<details><summary><strong>Or</strong> test <strong>locally</strong> with three Getting Started pages. <sub>(click to toggle details)</sub></summary>

1. Ensure you have all guide page templates available
    e.g. your `settings_local.py` has a `CMS_TEMPLATES` entry like:
    ```
    CMS_TEMPLATES = (
        # ...
        ('guide.html', 'Guide'),
        ('guides/getting_started.tam.html', 'Guide: Getting Started (TAM)'),
        ('guides/getting_started.v2.html', 'Guide: Getting Started (v2)'),
        ('guides/data_transfer.html', 'Guide: Data Transfer'),
        ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
    )
    ```
2. Create pages from each of these templates:
    <details><summary>"Guide" (with specific content) <sub>(click to toggle content)</sub></summary>

    ```
    <h1>Test with &lt;hr&gt;</h1>
    <hr />
    <h2>I Am I Normal Heading, No Tricks</h2>
    <p>The &lt;hr&gt; above should match the trick border of the &lt;h2&gt; in the second section.</p>
    <h1>Test sans &lt;hr&gt;</h1>
    <h2>I Have a Trick Border Inside Me</h2>
    <p>The trick border in the &lt;h2&gt; above should match the &lt;hr&gt; in the first section.</p>
    ```

    </details>

    "Guide: Getting Started (TAM)"

    "Guide: Getting Started (v2)"

    (optional) "Guide: Data Transfer"

    (optional) "Guide: Globus Data Transfer"

</details>

### Steps

1. Test that each Guide page shares matching markup and styles for:
    1. first-level heading as `<h1>` (not `<h2>`)
    2. second-level heading as `<h2>` (not `<h3>`)
    3. `<h1>` (spacing, font size)
    4. `<h2>` (spacing, font size)
    5. the border between any `<h1>` and `<h2>` (spacing, font size) *
    6. `<p>` (spacing, font size)
2. Test typography page changes:
    1. Each "Headings _With Content Beneath_" heading (except "HEADING 5…") has lowercase word "with".
    2. The "Heading 4 with Paragraph" is not uppercase.

<sub>* The [Custom Guide (FP-1800)](https://apcd-qa.tacc.utexas.edu/help/custom-guide/) shows two ways to make borders: `<h2>` (standard) and `<hr>` (usage as a line between headings is deprecated).</sub>

Guide pages to test:
- [Custom Guide (FP-1800)](https://apcd-qa.tacc.utexas.edu/help/custom-guide/)
- [Getting Started (TAM)](https://apcd-qa.tacc.utexas.edu/help/getting-started-tam/)
- [Getting Started (v2)](https://apcd-qa.tacc.utexas.edu/help/getting-started-v2/)
- (optional) [Data Transfer](https://apcd-qa.tacc.utexas.edu/help/data-transfer-guide/)
- (optional) [Globus Data Transfer](https://apcd-qa.tacc.utexas.edu/help/globus-data-transfer-guide/)
- [Design System: Typography](https://apcd-qa.tacc.utexas.edu/design-system/typography/)

### Guides

https://user-images.githubusercontent.com/62723358/187236989-533eb238-f5f9-49c8-9f7f-2a667a22e9f4.mov

| Custom Guide | Get… Started, TAM | Get… Started, v2 | Data Transfer | Globus Data … |
| - | - | - | - | - |
| Custom Guide | Getting Started (TAM) | Getting Started (v2) | Data Transfer Guide | Globus Data Transfer Guide |

### Typography

Typography

## Notes

PR Review Notice: https://tacc-team.slack.com/archives/GQW4Q8HLG/p1661790478409139

Low priority, because the big changes have workarounds on APCD and the minor changes are negligible.

[I added redirects on the CMS (prod and staging) for `/tam/register`.](https://github.com/TACC/Core-CMS/pull/549#issuecomment-1230805609)

[FP-1801]: https://jira.tacc.utexas.edu/browse/FP-1801
[FP-1800]: https://jira.tacc.utexas.edu/browse/FP-1800
[FP-1793]: https://jira.tacc.utexas.edu/browse/FP-1793
[FP-1452]: https://jira.tacc.utexas.edu/browse/FP-1452
[FP-1802]: https://jira.tacc.utexas.edu/browse/FP-1802
[FP-1336]: https://jira.tacc.utexas.edu/browse/FP-1336
[FP-1335]: https://jira.tacc.utexas.edu/browse/FP-1335

### CODE REVIEW

**Review by rstijerina — APPROVED** [review]
So, adding redirects via the cms?

**Comment by wesleyboar** [comment]
> So, adding redirects via the cms?
> — @rstijerina 

Yes. Until [FP-1803](https://jira.tacc.utexas.edu/browse/FP-1803). I added one redirect each on Prod and Staging underneath new CMS page `/tam` "TAM(Redirects)".

| Staging | Prod |
| - | - |
| APCD (Staging) | APCD (Prod) |

File: taccsite_cms/templates/guides/getting_started.tam.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+      <h2>Account Holders</h2>
+
+      <p>If you already have an account, you may log in by clicking the <q class="nav-link-ref" href="/login"><i class="icon icon-user"></i> Log in</q> link at the top right of the page.</p>
+
+      <h2 id="create-account">Creating an Account</h2>
+
+      <p>Before you can access the Portal, you must first have an active account with TACC.</p>
+
+      <p>If you are a new TACC user, you may <a href="{{ site_uri.absolute_uri }}tam/register" target="_blank">sign up for a new TACC account</a>.</p>
```
Comment (wesleyboar) [inline taccsite_cms/templates/guides/getting_started.tam.html:26]: [Redirect created on APCD CMS (prod and staging).](https://github.com/TACC/Core-CMS/pull/549#issuecomment-1230805609)

**Review by fnets — APPROVED** [review]
LGTM, including the optional pages! Thanks!

**Pull request merged**